bagelbids/cli.py

```diff
@@ -122,3 +124,40 @@ def pheno(
 
     with open(output / "pheno.jsonld", "w") as f:
         f.write(json.dumps(context, indent=2))
+
+
+@bagel.command()
+def add_bids(
```

Comment:
optional toppings (?): 
why not make this a one-word subcommand like `add` or `imaging` or `bids`? afaik we don't have plans to have `remove_bids` or `update_bids`?

Reply:
yeah good point, changing it to "bids"